src/transaction/log_replication.cpp

```diff
@@ -130,9 +136,12 @@ namespace cublog
 
     // redo all records from current position (m_redo_lsa) until end_redo_lsa
 
+    m_perfmon_log_processing.start ();

```

Comment:
- Rename m_perfmon_log_processing => m_perfmon_redo_sync?
- Why not start & track only once per redo_upto call? Imo the important information is here is the total time spent on the sync execution. It can inform us whether the sync operations are a bottleneck.

Reply:
I was thinking more to log perf at the level of a single entry in the log record. Because - more or less - the other entries are also logged at the level of processing a single entry.